src/main/java/org/terasology/equipment/component/EquipmentEffectComponent.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.equipment.component;
+
+import org.terasology.entitySystem.Component;
+
+public class EquipmentEffectComponent implements Component {
+
+    //the duration for which the effect lasts
```

Comment:
I think in this case the single line version could work, just so long as it is javadoc instead of standard comment? 

`/** The duration for which the effect lasts */`